ext/awscrt_arginfo.h

```diff
@@ -126,6 +126,91 @@ ZEND_END_ARG_INFO()
 
 #define arginfo_aws_crt_credentials_provider_static_new arginfo_aws_crt_event_loop_group_new
 
+#define arginfo_aws_crt_signing_config_aws_new arginfo_aws_crt_last_error
```

Comment:
These two lines frighten me.  What is the connection between event_loop_group_new and static credentials provider new?  same with signing_config_aws_new.

Reply:
The connection is that they use the same arguments/types/order. This is generated code, and is part of the php build toolset.